<?xml version="1.0" encoding="utf-8"?>

<launch>
<!--=================================e2box_AHRS V5=========================-->
  <node pkg="local_pkg" type="ahrs_parser.py" name="ahrs_raw" respawn="true" output="screen"/>

<!--=================================GNSS_Parsing=========================-->
  <arg name="node_name" value="ublox_gps"/>
	    <arg name="param_file_name" value="zed-f9p"/>
	    <arg name="output" value="screen"/>
	    <arg name="respawn" value="true"/>
	    <arg name="respawn_delay" value="30"/>
	    <arg name="clear_params" value="false"/>

  <node pkg="ublox_gps" type="ublox_gps" name="$(arg node_name)"
        output="$(arg output)" 
        clear_params="$(arg clear_params)"
        respawn="$(arg respawn)" 
        respawn_delay="$(arg respawn_delay)">
    <rosparam command="load" 
              file="$(find ublox_gps)/config/$(arg param_file_name).yaml" />
  </node>

<!--=================================Localization=========================-->
  <node pkg="local_pkg" type="time_sync.py" name="time" respawn="true"/>
  <node pkg="local_pkg" type="encoder.py" name="Encoder" respawn="true"/>
</launch>
